Source/EasyNetQ/AdvancedPublishConfiguration.cs

```diff
@@ -0,0 +1,43 @@
+namespace EasyNetQ
+{
+    public interface IAdvancedPublishConfiguration
```

Comment:
I am a little confused by the presence of `IAdvancedPublishConfiguration` and `IPublishConfiguration`. Both are for publish but what is responsibility of each? Moreover looking at `IAdvancedPublishConfiguration` name one can think that it is superset of `IPublishConfiguration`.